scripts/CeladonCity.asm

```diff
@@ -4,47 +4,47 @@ CeladonCity_Script:
 	ResetEvent EVENT_67F
 	ret
 
-CeladonCity_TextPointers:
-	dw CeladonCityText1
-	dw CeladonCityText2
-	dw CeladonCityText3
-	dw CeladonCityText4
-	dw CeladonCityText5
-	dw CeladonCityText6
-	dw CeladonCityText7
-	dw CeladonCityText8
-	dw CeladonCityText9
-	dw CeladonCityText10
-	dw CeladonCityText11
-	dw PokeCenterSignText
-	dw CeladonCityText13
-	dw CeladonCityText14
-	dw CeladonCityText15
-	dw CeladonCityText16
-	dw CeladonCityText17
-	dw CeladonCityText18
+	def_text_pointers CeladonCity
+	dw_const CeladonCityLittleGirlText,    TEXT_CELADONCITY_LITTLE_GIRL
+	dw_const CeladonCityGramps1Text,       TEXT_CELADONCITY_GRAMPS1
+	dw_const CeladonCityGirlText,          TEXT_CELADONCITY_GIRL
+	dw_const CeladonCityGramps2Text,       TEXT_CELADONCITY_GRAMPS2
+	dw_const CeladonCityGramps3Script,     TEXT_CELADONCITY_GRAMPS3
+	dw_const CeladonCityFisherText,        TEXT_CELADONCITY_FISHER
+	dw_const CeladonCityPoliwrathScript,   TEXT_CELADONCITY_POLIWRATH
+	dw_const CeladonCityRocket1Text,       TEXT_CELADONCITY_ROCKET1
+	dw_const CeladonCityRocket2Text,       TEXT_CELADONCITY_ROCKET2
+	dw_const CeladonCityTrainerTips1Text,  TEXT_CELADONCITY_TRAINER_TIPS1
+	dw_const CeladonCitySignText,          TEXT_CELADONCITY_SIGN
+	dw_const PokeCenterSignText,           TEXT_CELADONCITY_POKECENTER_SIGN
+	dw_const CeladonCityGymSignText,       TEXT_CELADONCITY_GYM_SIGN
+	dw_const CeladonCityMansionSignText,   TEXT_CELADONCITY_MANSION_SIGN
+	dw_const CeladonCityDeptStoreSignText, TEXT_CELADONCITY_DEPTSTORE_SIGN
+	dw_const CeladonCityTrainerTips2Text,  TEXT_CELADONCITY_TRAINER_TIPS2
+	dw_const CeladonCityPrizeExchangeSign, TEXT_CELADONCITY_PRIZEEXCHANGE_SIGN
+	dw_const CeladonCityGameCornerSign,    TEXT_CELADONCITY_GAMECORNER_SIGN
```

Comment:
Best to be consistent with the "SignText" suffix instead of just "Sign" (like just above with `CeladonCityGymSignText` etc).